akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowLogSpec.scala

```diff
@@ -28,7 +28,7 @@ class FlowLogSpec extends AkkaSpec("akka.loglevel = DEBUG") with ScriptedTest {
 
   "A Log" must {
 
-    val supervisorPath = "akka://FlowLogSpec/user/$a"
+    val supervisorPath = "akka://FlowLogSpec/user/StreamSupervisor-0"
```

Comment:
Right; related thought: should it not live under `/system`? For Persistence and Cluster actors like this live in the system namespace, should we move it (separate ticket)?